oas

```diff
@@ -490,11 +533,93 @@ components:
         archived: false
         name: 'Marta'
         public_identifier: 'DE 123456789'
-        farm_identifier: '674'
+        alternative_identifier: '674'
         pen_urn: 'urn:animal_service:FDBE6F55-6179-4B88-9177-4AFBED33ADAC'
-        date_birth: '2021-03-06T12:34:45Z'
         sex: 'f'
         species: 'cattle'
+
+    Event:
+      type: object
+      properties:
+        date:
+          type: string
+          format: date-time
+          description: date when this event happened
+        category:
+          $ref: '#/components/schemas/EventCategory'
+        animal_urn:
+          $ref: '#/components/schemas/URN'
+        payload: 
+          oneOf:
+            - description: birth event
+              type: object
+              properties:
+                mother_public_identifier:
+                  type: string
+                father_public_identifier:
+                  type: string
+            - description: arrival event
+              type: object
+              properties:
+                 arrival_reason:
+                  type: string
+                  description: reason why this animal arrives on the farm, **todo** extend enum
+                  enum:
+                    - 'buyed'
+            - description: departure event
+              type: object
+              properties:
+                 departure_reason:
+                  type: string
+                  description: reason why this animal arrives on the farm, **todo** extend enum
+                  enum:
+                    - 'death'
+                    - 'sold'
+                    - 'slaughered'
+            - description: milk-report event
+              type: object
+              properties:
+                milk:
+                  type: number
+                  description: amount of milk given in kg
+                milkFat:
+                  type: number
+                  description: in %
+                milkProtein:
+                  type: number
+                  description: in %
+    AnimalWeight:
+      type: object
+      required:
+        - animal_urn
+        - average_weight
+        - median_weight
+        - date
+      properties:
+        animal_urn:
+          type: string
+          example: urn:animal_service:FDBE6F55-6179-4B88-9177-4AFBED33ADAC
+          description: Uniform resource name of an object with the syntax urn:<service_id>:<internal_id>. <service_id> must be the urn of a valid and active ATLAS service.
+        average_weight:
+          description: average weight of the animal in kg
+          type: number
+        calculated_weight:
+          description: calculated weight of the animal in kg
+          type: number
+        daily_weight_gain:
+          description: daily gain of weight in kg
+          type: number
+        date:
```

Comment:
What is the meaning of this date?

Reply:
the hole object is a calculated value because on a real scale in a barn the animals walk multiple times a day over it and the scale calculate some daily values which are much more meaningful than the single measurements.
so date is the day for the calculation